delivery_carrier_service_level/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2018 Simone Rubino - Agile Business Group
```

Comment:
Was this originally made by SImone Rubino? In that case he/she has to be added to the authors. Else this line should be changed.